components/vf-componenet-rollup/index.scss

```diff
@@ -16,7 +16,6 @@
 @import 'vf-global-custom-properties.scss';
 
 @import 'vf-global-variables.scss';
-@import '_vf-variables.shame.scss';
 @import 'vf-global-variables.scss';
```

Comment:
I cannot remember why we've got this in twice?

Reply:
Hrm, looks like merge conflict gone awry.